src/datasources/human-description-api/human-description-api.service.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { HumanDescriptionApi } from './human-description-api.service';
+
+describe('HumanDescriptionAPI', () => {
+  it('should return descriptions from json', () => {
+    const humanDescriptionApi = new HumanDescriptionApi();
+
+    const descriptions = humanDescriptionApi.getDescriptions();
+
+    expect(descriptions['function transfer(address, uint256)']).toEqual(
```

Comment:
In its current state its not really a useful test since it just checks that the json import is returned by the `getDescriptions` call so we might as well change it to something like:

```
import ContractDescriptions from './json'

const descriptions = humanDescriptionApi.getDescriptions();
expect(descriptions).toEqual(ContractDescriptions);
```

Or I can remove it until we extend the api service. Wdyt?

Reply:
Since it's still a valid test I'd consider extending/refactoring it after merging the PR 👍 

The scope is already quite big for this one 😅

Reply:
Yeah, no need to tackle on the PR, I was just noticing this to see if we are aligned in the case of a future improvement 👍